en/docs/learn/api-microgateway/deploy-apis-in-privatejet-mode.md

```diff
@@ -0,0 +1,321 @@
+
+    <html><div class="admonition note">
+      <p class="admonition-title">Note</p>
+      <p>If the APIM deployed in a kubenretes cluster Master URL and serviceaccount token are not required to configure. </p>
+      </div>
+    </html>
+    
+5. Adding configurations of clusters.
+
+    For the super tenant users the cluster configurations should be added to the `<API-M_HOME>/repository/conf/deployment.toml` file as follows.
+    
+    ``` json tab="Format"
+    [[ContainerMgtClusterConfig]]
+    type = "<cloud-environment-type>"
+    clusterName = "<cluster-name>"
+    displayName = "<cluster-name-to_display>"
+    properties.Replicas = "<number-of-replicas>"
+    properties.AccessURL = "<access-URL>"
+    properties.MasterURL = "<Master-URL-of-clsuter>"
+    properties.SAToken = "<serviceacccount-token>"
+    properties.Namespace = "<namespcae>"
+    ```
+
+    ``` json tab="Example"
+    [[ContainerMgtClusterConfig]]
+    type = "Kubernetes"
+    clusterName = "docker-desktop"
+    displayName = "cluster-1"
+    properties.Replicas = 1
+    properties.AccessURL = "mgw.ingress.wso2.com"
+    properties.MasterURL = "https://kubernetes.docker.internal:6443"
+    properties.SAToken = "eyJhbGciOiJSUzI1NiIsImtpZCI6Ijg2UlYySEFkaVBOVG4zZ3FhWERXQUxQVUQ0bjlzOXE3dFhpbUZNZlFiRjQifQ.eyJpc3MiOiJrdWJlcm5ldGVzL3NlcnZpY2VhY2NvdW50Iiwia3ViZXJuZXRlcy5pby9zZXJ2aWNlYWNjb3VudC9uYW1lc3BhY2UiOiJ3c28yIiwia3ViZXJuZXRlcy5pby9zZXJ2aWNlYWNjb3VudC9zZWNyZXQubmFtZSI6IndzbzJhbS1wYXR0ZXJuLTEtc3ZjLWFjY291bnQtdG9rZW4tZGR0OTkiLCJrdWJlcm5ldGVzLmlvL3NlcnZpY2VhY2NvdW50L3NlcnZpY2UtYWNjb3VudC5uYW1lIjoid3NvMmFtLXBhdHRlcm4tMS1zdmMtYWNjb3VudCIsImt1YmVybmV0ZXMuaW8vc2VydmljZWFjY291bnQvc2VydmljZS1hY2NvdW50LnVpZCI6Ijc3NmRmMjFkLTA3NjItNDM2Zi05ZDIwLTZlYzFkODMxYzc1NSIsInN1YiI6InN5c3RlbTpzZXJ2aWNlYWNjb3VudDp3c28yOndzbzJhbS1wYXR0ZXJuLTEtc3ZjLWFjY291bnQifQ.YwlSgnSnwS7BNP0q0iDHsjlq_r3RhobG-SrduuEi35VXNfipPsK3UVJcCBmDKQOzKAxRG9415h2pr3cS-QsM6PR_UU2UWlXiUO-3UtSMqVY48Ek_1wdfmfCkiE7IOeua_58CV15tXtMLom0Oh27nkmWGfTHVvImQnqho4nv26BKZH4vKWgkw0HpMt73KHdN6SlVMK0cynktO5H-2A4V2rh-uL-OUXCLON_sBypVoPc9PAMxHo-bUe0SqBPiM0SiALu-0-J6dBcwbzPb5g-yUZmHmtuw3O32C304Hgfr-4Dui3X5DSKBSeqlrjjjfrvMVNug0J4JKk3bJ56h0OgujFw"
+    properties.Namespace = "default"
+    ```
+         
+       Tenant users, can edit the cluster configurations included in `tenant-conf.json` file. Go to the carbon console and edit the tenant-conf.json file in `/_system/config/apimgt/applicationdata/` path.
+       
+    ``` json tab="Format"
+    "ContainerMgt": [
+      {
+        "Type": "Kubernetes",
+        "ContainerMgtInfo": [
+        {
+          "ClusterName": "",
+          "DisplayName": "",
+          "Properties": {
+            "MasterURL": "",
+            "AccessURL": "",
+            "SAToken": "",
+            "Namespace": "default",
+            "Replicas": 1,
+            "BasicSecurityCustomResourceName": "",
+            "OauthSecurityCustomResourceName": "",
+            "JWTSecurityCustomResourceName": ""
```

Comment:
Why BasicSecurityCustomResourceName, OauthSecurityCustomResourceName and JWTSecurityCustomResourceName are not there in deployment.toml?

Reply:
removed from the conf